packages/dso-toolkit/gulp/build-svg-spritesheet.js

```diff
@@ -13,135 +13,136 @@ import { sassTransformer } from "./transformers/sass.transformer.js";
-
               variants.forEach((variant, index) => {
-                const x = (position + index) * (canvas + gutter) + gutter / 2;
-                const svg = symbol.clone().removeAttr("id");
-                $(svg).each((_index, element) => (element.tagName = "svg"));
-
-                if (variant.color) {
-                  svg.find('[fill="currentColor"]').attr("fill", variant.color);
-                }
+                const x = (position + index) * (canvasSize + gutterSize) + gutterSize / 2;
 
                 const view = $("<view>")
-                  .attr("id", `img-${variant.selector}`)
-                  .attr("viewBox", [x, 0, canvas, canvas].join(" "));
+                  .attr("id", `${variant.selector}`)
+                  .attr("viewBox", [x, 0, canvasSize, canvasSize].join(" "));
+
+                const svgElement = $("<svg>").attr("id", variant.selector).attr("viewBox", $(symbol).attr("viewBox"));
 
-                symbol.before(view);
+                $(symbol)
+                  .children()
+                  .clone()
+                  .each((_, child) => {
+                    if (variant.color && $(child).attr("fill") === "currentColor") {
+                      $(child).attr("fill", variant.color);
+                    }
+                    svgElement.append(child);
+                  });
 
-                const g = $("<g>").attr("transform", `translate(${x})`).append(svg);
+                const group = $("<g>").attr("transform", `translate(${x})`).append(svgElement);
 
-                symbol.before(g);
+                $(symbol).before(view).before(group);
               });
 
-              return position + variants.length;
-            }, 0);
-
-            $(":root").attr(
-              "viewBox",
-              [
-                (positions * (canvas + gutter)) / 2 - (canvas + gutter) / 2 + gutter / 2,
-                0,
-                positions * (canvas + gutter) + gutter,
-                canvas,
-              ].join(" "),
-            );
-          },
-          parserOptions: { xmlMode: true },
-        }),
-      )
-      .pipe(prettier({ plugins: ["@prettier/plugin-xml"] }))
-      .pipe(rename("dso-icons.svg"))
-      .pipe(gulp.dest(distPath))
-      .on("end", resolve)
-      .on("error", reject);
-  });
+              $(symbol).remove();
+              position += variants.length;
+            }
+          });
+
+          // Calculate viewBox based on total width and height
+          const width = position * (canvasSize + gutterSize) + gutterSize;
+          $("svg:first-child").attr("viewBox", `0 0 ${width} ${canvasSize}`);
+        },
+        parserOptions: { xmlMode: true },
+      }),
+    )
+    .pipe(rename(spriteName))
+    .pipe(gulp.dest(distPath));
+}
+
+export async function buildSvgSpritesheet() {
+  // Generate both styled and unstyled spritesheets
+  await Promise.all([buildSprite(outPutNameIcons, false), buildSprite(outPutNameIconsWithVariants, true)]);
```

Comment:
`outPutNameIcons` en `outPutNameIconsWithVariants` worden alleen op deze plek gebruikt. Heeft het gebruik van deze consts dan toegevoegde waarde? Wat mij betreft hier `dso-icons.svg` en `di.svg` gebruiken.

Reply:
Je zou `false` in de `buildSprite(outPutNameIcons, false)`; immers `iconsWithStyles` defaults to false in `buildSprite`

Reply:
Helemaal mee eens, was nog wat aan het refactoren geslagen en dit is aan mijn aandacht ontglipt